src/main/resources/jupyter/init-actions.sh

```diff
@@ -17,6 +17,26 @@ set -e -x
 # Retry 4/5 exited 1, retrying in 16 seconds...
 # Retry 5/5 exited 1, no more retries left.
 #
+# Array for instrumentation
+# UPDATE THIS IF YOU ADD MORE STEPS:
+# currently the steps are:
+# START init,
```

Comment:
Ah sorry, this will conflict a bit with https://github.com/DataBiosphere/leonardo/pull/1034, where I split the init script into 2 to support custom Dataproc images. Hopefully it shouldn't be too bad to merge.